### PR TITLE
added a helper script that creates comparison markdown table

### PR DESCRIPTION
**Pull request by jakubczakon**
Added:
* helper script that creates a markdown file with a comparison table that shows a diff between to experiments
* auto-generated docs

Changed:
* moved `neptunecontrib.sync.with_json` to main `neptunecontrib` module for simplicity and consistency

### CODE REVIEW

File: neptunecontrib/create_experiment_comparison_comment.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                           api_token=arguments.api_token)
+    return project.get_leaderboard(id=arguments.experiment_ids)
+
+
+def get_experiment_data_by_tag(arguments):
+    project = neptune.init(project_qualified_name=arguments.project_name,
+                           api_token=arguments.api_token)
+
+    experiment_ids = []
+    for tag in arguments.tag_ids:
```
Comment (hubertjaworskineptune) [inline neptunecontrib/create_experiment_comparison_comment.py:67]: maybe
`experiment_id = [project.get_experiments(tag=tag)[0].id for tag in arguments.tag_ids]`

File: neptunecontrib/create_experiment_comparison_comment.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+
+def find_experiment_diff(df):
+    selected_cols, cleaned_cols = [], []
+    for col in df.columns:
+        for name in ['channel_', 'parameter_', 'property_']:
+            if name in col and not any(excluded_name in col for excluded_name in ['stderr', 'stdout']):
+                selected_cols.append(col)
+                cleaned_cols.append(col.replace(name, ''))
+    df = df[['id'] + selected_cols]
```
Comment (hubertjaworskineptune) [inline neptunecontrib/create_experiment_comparison_comment.py:81]: change name of assigned variable - shadowing is bad practice

File: neptunecontrib/create_experiment_comparison_comment.py [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Create a markdown file with an experiment comparison table.
+
+Get a diff between experimenst across metrics, parameters, and properties and save it to a file as a markdown table.
+
+Attributes:
+    tag_ids(list(str)): UUID tags of experiments you would like to compare. It works only for 2 experiments.
```
[the remark this answers is not in the document]
Comment (jakubczakon) [reply neptunecontrib/create_experiment_comparison_comment.py:22]: Those are actually UUID that I need to push somehow to Neptune and later retrieve experiments by them. 
Figured that tags are easiest (no properties in the `.get_leaderboard()` method). 

How would you change this?

File: neptunecontrib/create_experiment_comparison_comment.py [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+
+def get_experiment_data_by_id(arguments):
+    project = neptune.init(project_qualified_name=arguments.project_name,
+                           api_token=arguments.api_token)
+    return project.get_leaderboard(id=arguments.experiment_ids)
+
+
+def get_experiment_data_by_tag(arguments):
+    project = neptune.init(project_qualified_name=arguments.project_name,
```
[the remark this answers is not in the document]
Comment (jakubczakon) [reply neptunecontrib/create_experiment_comparison_comment.py:63]: Fair point, on it.

[the remark this answers is not in the document]
Comment (jakubczakon) [reply neptunecontrib/create_experiment_comparison_comment.py:63]: Although, if you look at the code `neptune.init` can never be run twice -> it's in the if elif blocks.
Knowing that would you still change it?

File: neptunecontrib/create_experiment_comparison_comment.py (hunk 1 above)
Comment (jakubczakon) [reply neptunecontrib/create_experiment_comparison_comment.py:67]: thanks

File: neptunecontrib/create_experiment_comparison_comment.py (hunk 2 above)
Comment (jakubczakon) [reply neptunecontrib/create_experiment_comparison_comment.py:81]: ok

**Pull request merged**